### DISCUSSION TITLE
Change port from 80 to anything

### DISCUSSION THREAD

**Discussion by dieter-erich** [arendst/Tasmota#18276] [Q&A]
Hi,
    I have installed two Tasmota plugs and can turn them on and off locally (via IP 10.0.0.4 and 10.0.0.12). For remote access, on the modem, I then set port forwarding 8080 > 80 and the mac address of plug 1, which works fine from the public address, when specifying port 8080. For the second plug I set port forwarding to 8081 > 80 (including the corresponding mac address) but this does not work. Surely, there might be a problem of the same local port (80) but I thought that the modem would sort that out via the specified mac addresses. I am a bloody beginner, can anybody please give me a hint!
Thanks, D-E

**Comment by barbudor** [comment]
I have a site with 5 of such forward and working without any issue
I also have source IP filtering to secure the access only from my main site as Tasmota is not secure enough.

So the problem is likely a problem on your router configuration
Note that on my router, forwarding is based on IP address not MAC which would be surprising

**Comment by dieter-erich** [comment]
Hi,

   thanks a lot! But I do not understand: the Tasmota device connects 
through port 80. So, if the mac address is not specified, port 
forwarding 8080 > 80 and 8081 > 80 will not distinguish the two devices! 
Maybe the port of the device can be changed to, say, 81, 82 etc. Then 
port forwarding would be unequivocal and easy and not requiring the mac 
address! I couldn't find out how this could be done!

  best

------------------------------------------------------------------------
Dieter Blaas,
Max Perutz Laboratories
Medical University of Vienna,
Inst. Med. Biochem., Vienna Biocenter (VBC),
Dr. Bohr Gasse 9/3,
A-1030 Vienna, Austria,
Tel: 0043 1 4277 61630,
Mobile: 0043 699 1942 1659
***@***.***
------------------------------------------------------------------------

**Reply by barbudor** [reply]
You should be able to specify the destination ip address like
Outside:8080 => 192.168.1.10:80
Outside:8081 => 192.168.1.11:80
I can't access the remote router now to make a screen shot but I'll try later

**Comment by Noschvie** [comment]
On a pfsense it looks like

![grafik](https://user-images.githubusercontent.com/48217146/227919939-2f43ebf6-df29-4c0c-8f58-412981c0841d.png)

**Comment by dieter-erich** [comment]
Hi! both of you, many thanks for your willingness to help! I have 
exactly the setup shown in the screenshot!

1) I define the rule (image 1 = Tasmota-10.0.0.12 with the forwarding 
name Tasmota2)

2) I apply it to the device with the indicated mac address (image 2).

I did the same with Tasmota 1 (just that I forward 8080 instead of 8081 
to 80) This one works, the 2nd not!

no avail! If I find out I'll post it!

On 2023-03-27 12:45, Norbert wrote:

> On a pfsense it looks like
> 
> [1]
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>

-- 
Dr. Dieter Blaas
Max F. Perutz Laboratories,
Inst. Med. Biochem.,
Med. Univ. Vienna
Dr. Bohr Gasse 9/3
A-1030 Vienna, Austria
Tel. 0043 1 4277 61630
Fax  0043 1 4277 9616

Links:
------
[1] 
https://user-images.githubusercontent.com/48217146/227919939-2f43ebf6-df29-4c0c-8f58-412981c0841d.png
[2] 
https://github.com/arendst/Tasmota/discussions/18276#discussioncomment-5439397
[3]

**Reply by barbudor** [reply]
I see no pictures from you

**Comment by lalo-uy** [comment]
It looks that you are mixin 2 diferent setups in the router:
First you assign a fixed ip to the mac of the devices.
Then you set up the redirect the 8080 abd 8081 to that fixed ips:80

If do not work try other browser. Chrome try to access with https not mater
the port , grrrrrgrgrgr. Firefox is more like to go ok.

**Reply by dieter-erich** [reply]
Thanks again! I have to check the accessibility of the device. It seems it is too far from the modem! Probably this is the reason for the problem!
